package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "search-parser",
+  "version": "1.0.0",
+  "description": "Search parser to simplify search param identification and matching",
+  "author": "Graham Batzler",
+  "scripts": {
+    "test": "mocha",
+    "lint": "dibslint --root=.",
+    "gitlint": "node_modules/.bin/dibslint --git --warnings"
```

Comment:
`npm` adds `node_modules/.bin/` to $PATH when you run scripts, so you should need to prefix your .bin binaries with it here.